features/sample.feature

```diff
@@ -15,15 +15,18 @@ Feature: Hacker News REST API validation
     Then the response status should be "200"
     And the JSON response root should be array
     When I grab "$[0]" as "id"
+    And I save the "$[0]" as "id"
```

Comment:
I understand that you add a 'save' step here because 'grab' would clear grabbed values after we send request? I would suggest instead to change the meaning of 'grab' to keep the value until you override it with another 'grab' or we finish our scenario. That way we have 1 single way to temporarily store values and don't confuse with 2 choices to store values.

E.g.
```
When I grab "$[0]" as "id"
...
When I grab "$[1]" as "id" // previous id overriden here and used by subsequent steps
```
For clearing 'grab' values you can use [scenario hooks](https://github.com/cucumber/cucumber/wiki/Hooks). What do you think?